docs/internals/contributing/writing-documentation.txt

```diff
@@ -256,6 +256,25 @@ If a function, attribute, etc. is added, it's also okay to use a
 We can simply remove the ``.. versionadded:: A.B`` annotation without any
 indentation changes when the time comes.
 
+Minimizing images
+-----------------
+
+Optimize image compression where possible.  For PNG files, try OptiPNG and AdvanceCOMP's ``advpng``:
+
+.. code-block:: sh
```

Comment:
"sh" -> "console"
One space after period and wrap lines at 79 characters please.